packages/components/src/components/FileUploader/useFileUploader.ts

```diff
@@ -88,6 +100,24 @@ export const useFileUploader = <TRootElement extends HTMLElement>(
 
   const { files: stateFiles } = state;
 
+  useEffect(() => {
+    if (value) {
```

Comment:
For lesbarhetens skyld. Kan dere lage en `isControlled`-const som kan brukes i denne if-sjekken og if-sjekkene lenger ned? Det vil gjøre det enklere å forstå hva formålet med `if`-ene er istedenfor å bare sjekke på "value".